scripts/scrabble.js

```diff
@@ -0,0 +1,48 @@
+const rl = require( 'readline-sync')
+
+getInput = () => {
+  let theLetters = Array.from( rl.question( 'What are the letters? ' ).toUpperCase() )
```

Comment:
Consider omitting the temporary variable declaration, and directly returning the calculation.  Since it would then be a single line return, you could omit the `return` statement and the braces:
```js
const getInput = () => 
  Array.from( rl.question( 'What are the letters? ' ).toUpperCase() )
```